homeassistant/helpers/entity.py

```diff
@@ -278,19 +279,27 @@ def schedule_update_ha_state(self, force_refresh=False):
         self.hass.add_job(self.async_update_ha_state(force_refresh))
 
     def remove(self) -> None:
-        """Remove entitiy from HASS."""
+        """Remove entity from HASS."""
         run_coroutine_threadsafe(
             self.async_remove(), self.hass.loop
         ).result()
 
     @asyncio.coroutine
     def async_remove(self) -> None:
-        """Remove entity from async HASS.
+        """Remove entitiy from async HASS.
```

Comment:
Writer's freedom :-P

I rebased at some point, this was corrected recently and I must have missed it